bigtable/hello_world.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# [START bigtable_hw_imports]
+require "google/cloud/bigtable"
+# [END bigtable_hw_imports]
+
+# [START bigtable_hw_connect]
+project_id = "YOUR_PROJECT_ID"
+table_id = "Hello-Bigtable"
+instance_id = "my-instance"
+column_family = "cf"
+column_qualifier = "greeting"
+
+bigtable = Google::Cloud::Bigtable.new project_id: project_id
```

Comment:
I'm open to leaving this as is, but generally the project_id can be inferred automatically via service account etc. What was your thought process for setting it explicitly?

Reply:
There is no specific reason but other languages hello_world example following same pattern to set project_id.